content/en/content-management/summaries.md

```diff
@@ -33,6 +33,35 @@ This is the first paragraph.
 This is the second paragraph.
 ```
 
+> [!NOTE]
+> Place the summary divider on its own line. Do not place it inline with other content.
+
+**Correct placement:**
```

Comment:
Per the documentation guidelines (https://gohugo.io/contribute/documentation/) do not use bold text in place of a heading or description term (dt).